sarif/include/gap/sarif/definitions.hpp

```diff
@@ -0,0 +1,2866 @@
+// Copyright 2024-present, Trail of Bits, Inc.
+
+#pragma once
+
+#include <cstdint>
+#include <forward_list>
+#include <optional>
+#include <string>
+#include <unordered_map>
+
+#include <nlohmann/json.hpp>
+
+namespace gap::sarif::definitions
+{
```

Comment:
add `using json = nlohmann::json;` to have cleaner API

Reply:
Done!